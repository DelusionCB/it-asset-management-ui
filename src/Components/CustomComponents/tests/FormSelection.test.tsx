import React from 'react';
import * as TestRenderer from 'react-test-renderer';
import FormSelection from '../FormSelection';
import '@testing-library/jest-dom'

describe('<FormSelection />', () => {
    let renderer: TestRenderer.ReactTestRenderer;

    // Render the component before tests
    beforeAll(() => {
        renderer = TestRenderer.create(
            <FormSelection disabled={false} onChange={jest.fn()} selections={['selection']}/>,
        );
    });
    describe('Snapshot', () => {
        it('renders correctly', () => {
            expect(renderer.toJSON()).toMatchSnapshot();
        })
    })
})
