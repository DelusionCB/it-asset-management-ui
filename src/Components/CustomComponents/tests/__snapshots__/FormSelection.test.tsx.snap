// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormSelection /> Snapshot renders correctly 1`] = `
<div
  className="selection-wrapper"
>
  <div
    className="selection-header"
  >
    <h1>
      Valitse arkistoinnin tyyppi
    </h1>
  </div>
  <div
    className="selection-radios"
  >
    <div>
      <div
        className="form-check"
      >
        <label
          className="form-check-label"
          htmlFor="selection-0"
        >
          radios.selection
        </label>
        <input
          aria-disabled={false}
          checked={false}
          className="form-check-input"
          disabled={false}
          id="selection-0"
          name="selection"
          onChange={[Function]}
          type="radio"
        />
      </div>
    </div>
  </div>
</div>
`;
